README.md

```diff
@@ -53,7 +67,7 @@ git merge upstream/main main
 ```
 Here, `main` is the branch name of your repo, `upstream` refers to the repo on the `schwallergroup`, and `upstream/main` to its main branch.
 
-### Step 5: Push Updates to the Forked Repository
+### Step 6: Push Updates to the Forked Repository
```

Comment:
Even with them modifying it in the browser?

Reply:
Modifying in the browser is fine for a README, but I can see them uploading files to repos through the upload function instead of using the command line if we don't introduce it at some point.

Reply:
here: https://github.com/schwallergroup/practical-programming-in-chemistry-exercises/tree/main/week_01#making-changes-and-committing-them